### ISSUE TITLE
Plugin specific command

### ISSUE THREAD

**Issue by pbogut** [mkusher/padawan.php#25]
I would like to be able to add a custom command to padawan.php plugin. As far as I can tell this is currently not possible.
Can we add something like special route, where we can pass plugin action?
I was thinking about a command that will dispatch event, which can be handled by the plugin.
What do you think? I'm happy to write a patch if you think that's a good idea.

**Comment by mkusher** [comment]
@pbogut yeah, it can be a good idea, but first of all, why do you need it?

**Comment by pbogut** [comment]
Ok, so I'm going back to my Magento plugin. Magento is using XML for many things including defining classes generated by a factory method. When XML file is changing it should be reloaded by the plugin. With custom plugin command it should be easy enough to trigger this action (manually, or by an auto command in vim on a buffer save).
Hopefully, that makes sense.

**Comment by mkusher** [comment]
The problem I see right now is structure and API of padawan, that's going to change soon(hope in March I'll have enough time). So, I'm afraid your plugin/change will be incompatible with the next padawan version. Of course, if you do it, I'll try to help in migration and so. 
Saying honestly I'm not even sure whether I'll break this thing
